### PR TITLE
Clean dependencies 

### PR DESCRIPTION
**Pull request by 0xGabi**
# 🦅 Pull Request

I decided to include this PR to clean a bit before moving to v7 and start the separation of packages.

Update to client `v0.8.2`

@macor161 would you mind take a look?

### CODE REVIEW

File: packages/aragon-cli/package.json [hunk 1]
```diff
@@ -67,11 +62,10 @@
     "go-platform": "^1.0.0",
     "http-server": "^0.11.1",
     "ignore": "^4.0.6",
-    "inquirer": "^6.4.1",
+    "inquirer": "^7.0.0",
     "internal-ip": "^4.3.0",
     "ipfs-api": "^26.1.2",
     "ipfs-http-client": "^33.0.1",
-    "js-sha3": "^0.8.0",
```
Comment (macor161) [inline packages/aragon-cli/package.json]: I think `js-sha3` is still used at a couple of places.

File: packages/aragon-cli/src/cli.js [hunk 2]
```diff
@@ -35,7 +35,6 @@ cmd.middleware(MIDDLEWARES)
 
 cmd.alias('env', 'environment')
 cmd.alias('h', 'help')
-cmd.alias('v', 'version')
```
Comment (macor161) [inline packages/aragon-cli/src/cli.js]: Do we have a good reason to remove this?

Comment (0xGabi) [reply packages/aragon-cli/src/cli.js]: Oh this cause what we discuss here: https://github.com/aragon/aragen/pull/79#discussion_r332967208

File: packages/aragon-cli/package.json (hunk 1 above)
Comment (0xGabi) [reply packages/aragon-cli/package.json]: Good catch forgot to update to use `web3.utils`

File: packages/aragon-cli/src/cli.js (hunk 2 above)
Comment (macor161) [reply packages/aragon-cli/src/cli.js]: Ah ok I wasn't aware of the issue between `aragon -v` and `aragon devchain -v`, thanks 🙂 
However `aragon devchain -v` doesn't seem to be working either, is this normal?

Comment (0xGabi) [reply packages/aragon-cli/src/cli.js]: Oh sorry about that, it was cause I needed to update aragen first. I did that yesterday

**Pull request merged**